olp-cpp-sdk-dataservice-write/tests/CMakeLists.txt

```diff
@@ -107,6 +107,7 @@ else()
     target_include_directories(olp-cpp-sdk-dataservice-write-tests
     PRIVATE
         ${${PROJECT_NAME}_SOURCE_DIR}/src
+        ${PROJECT_SOURCE_DIR}/../olp-cpp-sdk-authentication/src
```

Comment:
Why is this needed? This usually indicates we have an API break?

Reply:
removed